src/sidebar.cpp

```diff
@@ -458,9 +458,18 @@ void player::disp_status( const catacurses::window &w, const catacurses::window
     }
 
     // display mood smiley
-    mvwprintz( w, sideStyle ? 6 : 3, sideStyle ? getmaxx( w ) - 2 : 0, col_morale,
-               morale_emotion( morale_cur, fc,
-                               get_option<std::string>( "MORALE_STYLE" ) == "horizontal" ) );
+    bool mood_style_hor = get_option<std::string>( "MORALE_STYLE" ) == "horizontal";
```

Comment:
Could be const.

Reply:
apologies, this one slipped me by unnoticed.
i use vertical smiley, and was busy on other PR.
i should check the issues tab more often. 
otoh, this is probably already fixed in the sidebar_info3 waiting for the merge.
but i would have provided interim fix in the same time as for the power value placement one.

Reply:
> apologies, this one slipped me by unnoticed.
> i use vertical smiley, and was busy on other PR.
> i should check the issues tab more often.
> otoh, this is probably already fixed in the sidebar_info3 waiting for the merge.
> but i would have provided interim fix in the same time as for the power value placement one.
Its np. I called out your fix in #27809 in the summary up top but figured it was good to go ahead and fix this before 0.D in case the discussion on your PR lasts a bit